src/app/components/OffersSearch/Offers/Stocks/Stocks.tsx

```diff
@@ -0,0 +1,52 @@
+import React from "react"
+
+import {
+  FORMAT_DD_MM_YYYY_HH_mm,
+  toISOStringWithoutMilliseconds,
+} from "utils/date"
+import { StockType } from "utils/types"
+
+import { formatLocalTimeDateString } from "../../../../../utils/timezone"
+
+const stockLineInfo = (
+  stock: StockType,
+  venueDepartmentCode: string
+): string => {
+  const stockBeginningDate = new Date(stock.beginningDatetime)
+  const stockBeginningDateISOString =
+    toISOStringWithoutMilliseconds(stockBeginningDate)
+  const stockLocalBeginningDate = formatLocalTimeDateString(
+    stockBeginningDateISOString,
+    venueDepartmentCode,
+    FORMAT_DD_MM_YYYY_HH_mm
+  )
+
+  const stockPrice: string = new Intl.NumberFormat("fr-FR", {
+    style: "decimal",
+    currency: "EUR",
+  }).format(stock.price)
+
+  return `${stockLocalBeginningDate}, ${stockPrice} €`
+}
+
+export const Stocks = ({
+  stocks,
+  venuePostalCode,
+}: {
+  stocks: StockType[];
+  venuePostalCode: string;
+}): JSX.Element => {
+  const listStocks = stocks.map((stock: StockType) => (
+    <li key={stock.id}>
+      {stockLineInfo(stock, venuePostalCode)}
```

Comment:
```suggestion
      {generateStockInformation(stock, venuePostalCode)}
```
Voire `print` vu que c'est finalement une visualisation des données du stock :P